local-data-platform/nyc_yellow_taxi.py

```diff
@@ -23,3 +24,23 @@
 
 # Verify if the catalog is set up correctly
 print("Catalog set up successfully:", catalog)
+
+'''
+CLI COMMAND to grab data -
```

Comment:
Is this for local testing ?

Reply:
@mrutunjay-kinagi yes. I ll added it to the modules read me like iceberg.